### PR TITLE
Fix queries in enumeration

### PR DESCRIPTION
**Pull request by manmolecular**
**Done:**
* Delete false and empty queries
* Fix trouble queries for Shodan
* Fix some queries for Google  

**Issues:**
* There are still some problems with Shodan queries for Huawei and ZTE (too many results that include routers and other network devices - difficult to identify unique SD-WAN information)

### CODE REVIEW

**Comment by manmolecular** [comment]
- Replace Juniper queries by contrail and contrail-control Juniper SD-WAN services

File: enumeration.md [hunk 1]
```diff
@@ -209,8 +199,7 @@
 * inurl:"/salsa/salsa_portal/"
 
 ### Shodan
-* ssl:SALSA
-* "Server: Apache"+title:SALSA+port:443
+* "Server: Apache"+title:"SALSA"*"Login" port:443
```
Comment (dnkolegov) [inline enumeration.md]: Please also add `ssl: SALSA Portal`

File: enumeration.md [hunk 2]
```diff
@@ -231,3 +219,7 @@
```
Comment (dnkolegov) [inline enumeration.md]: This does not work. About 400 000 entities.

File: enumeration.md [hunk 3]
```diff
@@ -162,23 +155,20 @@
...
+* ssl:"Nuage"*"Networks"
 
 ### Censys
 * 80.http.get.title: "Nuage"
 
 ## Juniper Networks Contrail SD-WAN
 ### Shodan
-* "Juniper Networks, Inc."
-* "server: Embedthis Appweb 3.2.3"
+* contrail
```
Comment (dnkolegov) [inline enumeration.md]: Does not work

File: enumeration.md [hunk 4]
```diff
@@ -162,23 +155,20 @@
...
+* contrail-control
```
Comment (dnkolegov) [inline enumeration.md]: Does not work

File: enumeration.md (hunk 1 above)
Comment (manmolecular) [reply enumeration.md]: Okay, thanks.

File: enumeration.md [hunk 5]
```diff
@@ -102,13 +101,7 @@
 * 80.http.get.body_sha256: 1d10f43efe5e0da430042178c7c8040d011bd5461279c5006ddabf867aae96cf
 
 ## Riverbed SteelConnect
```
[the remark this answers is not in the document]
Comment (manmolecular) [reply enumeration.md]: I had some problems with them (in CLI/Web Shodan), but it was temporary decision and now i return some of them - another queries were replaced by better versions. But basically everything in it's place now.

File: enumeration.md [hunk 6]
```diff
@@ -219,7 +208,6 @@
 
 ## Huawei SD-WAN
 ### Shodan
-* ssl:"O=Huawei"
 * "Agile Controller"
```
[the remark this answers is not in the document]
Comment (manmolecular) [reply enumeration.md]: I've got some kind of problem with Huawei Shodan queries right now and I try to solve it. I found [Huawei Agile Controller](http://e.huawei.com/en/products/enterprise-networking/sdn-controller/agile-controller?ic_source=fhc17), which is used with SD-WAN Huawei solutions, so, i think, we can use it somehow in our Shodan queries.

File: enumeration.md (hunk 2 above)
Comment (manmolecular) [reply enumeration.md]: I'm working on it, thanks.

File: enumeration.md (hunk 4 above)
Comment (manmolecular) [reply enumeration.md]: Okay, I will try to fix it. Simply put, the main idea was in searching of Juniper [Contrail SD-WAN service](https://www.juniper.net/us/en/products-services/sdn/contrail/contrail-sd-wan/) which is used with Juniper SD-WAN solutions.

**Pull request merged**